packages-ts/gauntlet-terra-cw-plus/src/commands/multisig.ts

```diff
@@ -1,16 +1,19 @@
 import { Result } from '@chainlink/gauntlet-core'
 import { logger, prompt } from '@chainlink/gauntlet-core/dist/utils'
 import { TerraCommand, TransactionResponse } from '@chainlink/gauntlet-terra'
-import { AccAddress, MsgExecuteContract, MsgSend } from '@terra-money/terra.js'
+import {
+  AccAddress,
+  MsgExecuteContract,
+  MsgSend,
+  MsgUpdateContractAdmin,
+  MsgMigrateContract,
+} from '@terra-money/terra.js'
 import { isDeepEqual } from '../lib/utils'
 import { fetchProposalState, makeInspectionMessage } from './inspect'
 import { Vote, Cw3WasmMsg, Action, State, Cw3BankMsg } from '../lib/types'
 
-type ProposalAction = (
-  signer: AccAddress,
-  proposalId: number,
-  message: MsgExecuteContract | MsgSend,
-) => Promise<MsgExecuteContract>
+type TerraMessage = MsgExecuteContract | MsgSend | MsgUpdateContractAdmin | MsgMigrateContract
```

Comment:
Isn't a type like this already available in `'@terra-money/terra.js'`?

Reply:
Yeah you are right. Updating it to support every Message type